application/org.openjdk.jmc.flightrecorder.flameview/src/main/java/org/openjdk/jmc/flightrecorder/flameview/FlameGraphJSONMarshaller.java

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright (c) 2020, Datadog, Inc. All rights reserved.
```

Comment:
Per OCA, Oracle copyrights will need to be in all files, and first. See other files.

Reply:
I have no idea how/why any of this works 😃